v19.2/performance.md

```diff
@@ -0,0 +1,101 @@
+
+### Sysbench
+
+[Sysbench](https://github.com/akopytov/sysbench) is a popular tool that allows for basic throughput and latency testing. Cockroach Labs prefers the more complex TPC-C, as discussed above, but Sysbench is a reasonable alternative for understanding basic throughput and latency across different databases. In the meantime, know that the numbers below were generated from a three-node cluster of AWS `c5d.9xlarge` VMs run across AWS’s us-east-1 region (availability zones a, b, and c) against Sysbench’s oltp_insert and oltp_point_select workloads.
+
+## Scale
+
+TPC-C provides the most realistic and objective measure for OLTP performance at various scale factors, and CockroachDB can hit an incredible 631K tpmC with 50,000 warehouses! That's **98% of the maximum possible throughput per minute for TPC-C 50K**.
+
+Comparing CockroachDB's unofficial TPC-C results to Amazon Aurora RDS's last published metrics from AWS re:Invent 2017, CockroachDB is now 50 times more scalable than Amazon Aurora, supporting 25 billion rows and more than 4 terabytes of frequently accessed data:
+
+<img src="{{ 'images/v19.2/tpcc50k.png' | relative_url }}" alt="TPC-C 50,000" style="max-width:100%" />
+
+                                   | CockroachDB       | Amazon Aurora   
+-----------------------------------|-------------------|--------------
+Max Throughput                     | 631851 tpmC       | 12582 tpmC     
+Max Warehouses with Max Efficiency | 50000 Warehouses  | 1000 Warehouses
+Max Number of Rows                 | 24.9B             | 0.499B         
+Max Unreplicated Data              | 4TB               | 0.08TB         
+Machine type                       | c5d.4xlarge       | r3.8xl         
+
+Unlike Amazon Aurora, CockroachDB achieves this performance in [`SERIALIZABLE` isolation](demo-serializable.html), the strongest isolation level in the SQL standard. Like many other databases, Aurora selectively degrades isolation levels for performance, leaving workloads susceptible to fraud and data loss.
+
+To learn more about our comparison with Amazon Aurora, see this [blog post](https://www.cockroachlabs.com/blog/cockroachdb-2dot1-performance/).
+
+<!-- To try this out for yourself on your laptop [visit this docs page](https://www.cockroachlabs.com/docs/v19.1/training/performance-benchmarking.html#main-content). Or, if you want to reproduce the full results of TPC-C [visit this page](https://www.cockroachlabs.com/guides/cockroachdb-performance/). -->
+
+### Linear Scaling
+
+CockroachDB has **no theoretical scaling limit** and, in practice, can achieve near-linear performance passed more than 256 nodes. Because the TPC-C results above reflect leaps in scale, to test linear scaling, Cockroach Labs ran a simple benchmark named KV 95 (95% point reads, 5% point writes, all uniformly distributed) on AWS `c5d.4xlarge` machines:
+
+<img src="{{ 'images/v19.2/linearscale.png' | relative_url }}" alt="CRDB Linear Scale" style="max-width:100%" />
```

Comment:
It'd be good to note somewhere the different client concurrencies. They're in the spreadsheet.

Reply:
We will add a table here.